apps/etterlatte-fordeler/src/main/kotlin/fordeler/Fordeler.kt

```diff
@@ -92,6 +104,28 @@ internal class Fordeler(
             hendelseGyldigTil = get(SoeknadInnsendt.hendelseGyldigTilKey).textValue().let(OffsetDateTime::parse)
         )
 
+    fun lagStatistikkMelding(packet: JsonMessage, fordelerResultat: FordelerResultat, sakType: SakType): String? {
+        val (resultat, ikkeOppfylteKriterier) = when (fordelerResultat) {
+            FordelerResultat.GyldigForBehandling -> true to null
+            is FordelerResultat.IkkeGyldigForBehandling -> false to fordelerResultat.ikkeOppfylteKriterier
+            else -> {
```

Comment:
er det ikke bedre å type ut den siste typen `UgyldigHendelse` istedet for å ha else?

Reply:
[7146909](https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/992/commits/7146909604ea12a76c2642ddd15e13593a273af8)